### DISCUSSION TITLE
7월25일 오전 기준으로 타임 2000 막혔네요

### DISCUSSION THREAD

**Discussion by ranware** [changdoc/naver-vaccine-macro#101] [Q&A]
7월25일 오전 기준으로 타임 2000 막혔네요

**Comment by murokim** [comment]
저는 잘되고 있습니다.

**Reply by ranware** [reply]
그런가요... 전 잠시 후 다시 시도하라고,,,

**Reply by MKK425** [reply]
> 그런가요... 전 잠시 후 다시 시도하라고,,,

창 여러개 켜놓고 하면 매크로로 감지해 1시간동안 안되는 걸로 알고 있습니다.

**Reply by ranware** [reply]
> > 그런가요... 전 잠시 후 다시 시도하라고,,,
> 
> 창 여러개 켜놓고 하면 매크로로 감지해 1시간동안 안되는 걸로 알고 있습니다.

지금 창 한개만 돌렸는 데.... 저만 이상한가보네요....

**Reply by MKK425** [reply]
> > > 그런가요... 전 잠시 후 다시 시도하라고,,,
> > 
> > 
> > 창 여러개 켜놓고 하면 매크로로 감지해 1시간동안 안되는 걸로 알고 있습니다.
> 
> 지금 창 한개만 돌렸는 데.... 저만 이상한가보네요....

저는 아직 작동 됩니다. 2000 기준으로요

다른 IP에서도 같은 아이디로 시도하면 매크로로 감지하는걸로 알아요.

**Comment by OneTop4458** [comment]
https://github.com/changdoc/naver-vaccine-macro/discussions/51
